ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Settings.hs

```diff
@@ -0,0 +1,336 @@
+
+  downloadLogsLabel <- createTwoLinesLabel "EXPORT LOGS" downloadLogsText
+  downloadLogsButton <- QPushButton.newWithText ("Download logs" ::  String)
+  addRow supportLayout downloadLogsLabel downloadLogsButton
+  addSeparator supportLayout
+
+
+  let enableReportsText = "Share anonymous usage and diagnostic data to help \
+                          \improve Ariadne Wallet, services and security features."
+  enableReportsLabel <- createTwoLinesLabel "AUTOMATIC REPORTS" enableReportsText
+  enableRepots <- QCheckBox.new
+  addRow supportLayout enableReportsLabel enableRepots
+  addSeparator supportLayout
+
+  let analyticsText = "Enable analytics of anonymous data to help improve the user experience. \
+                      \This includes the operating system, language, firmware versions and the numbebr of added accounts."
+  analyticsLabel <- createTwoLinesLabel "ANALYTICS" analyticsText
+  hasAnalytics <- QCheckBox.new
+  addRow supportLayout analyticsLabel hasAnalytics
+  addSeparator supportLayout
+
+  let resetText = "Erase all Ariadne Wallet data stored on your computer, \
+  \including your wallets, accounts, transaction history and settings."
+  resetLabel <- createTwoLinesLabel "RESET ARIADNE" resetText
+  resetButton <- QPushButton.newWithText ("Reset" :: String)
+  addRow supportLayout resetLabel resetButton
+
+  for_ [faqButton, clearCacheButton, reportButton, downloadLogsButton, resetButton] $ \w ->
+    QWidget.setSizePolicyRaw w Maximum Fixed
+
+  for_ [enableRepots, hasAnalytics] $ \w ->
+    QWidget.setSizePolicyRaw w Maximum Fixed
+
+  for_ [clearCacheButton, reportButton, downloadLogsButton] $ \b ->
+    setProperty b ("styleRole" :: Text) ("rightPaneSettingsButton" :: Text)
+
+  void $ setProperty resetButton ("styleRole" :: Text) ("resetSettingsButton" :: Text)
+
+  for_ [faqButton, clearCacheButton, reportButton, downloadLogsButton, resetButton] $ \b ->
+    QWidget.setEnabled b False
+
+  for_ [hasAnalytics, enableRepots] $ \b ->
+    QWidget.setEnabled b False
+
+  QWidget.setLayout supportWidget supportLayout
+
+  let ss = SupportSettings{..}
+  return (supportWidget, ss)
+
+showWidget :: Settings -> QWidget.QWidget -> QPushButton.QPushButton -> IO()
+showWidget Settings{..} widget button = do
+  for_ [generalButton, aboutButton, supportButton] $ \w ->
+    QAbstractButton.setChecked w False
+  QAbstractButton.setChecked button True
+  for_ [generalWidget, aboutWidget, supportWidget] $ \w ->
+    QWidget.setVisible w False
+  QWidget.setVisible widget True
+  QWidget.adjustSize settings
+
+showSettingsWindow :: UI Settings ()
+showSettingsWindow = do
+  settings <- view settingsL
+
+  liftIO $ QWidget.show settings
+
+
+createTwoLinesLabel :: String -> String -> IO QWidget.QWidget
+createTwoLinesLabel line1 line2 = do
+  widget <- QWidget.new
+  layout <- QVBoxLayout.new
+  label1 <- QLabel.newWithText("<b>" ++ line1 ++ "</b>" :: String)
```

Comment:
1. Space
2. I think we prefer using `<>` for string concatenation
3. No need to do `:: String`, it's string already do to function's type signature